### ISSUE TITLE
Fix goload.pro series identifiers

### ISSUE THREAD

**Issue by m-kuzmin** [m-kuzmin/ani-tui#6]
### Expected behavior
An ID obtained from the search should be usable to get to other episodes in the series

### Actual behavior
Some series have inconsistent links like:
```
goload.pro/videos/pokemon-xyz-episode-1
                  ~~~~~~~~~~~
goload.pro/videos/pokemon-xy-z-episode-30
                          ^^^^
```
So you cannot reach all episodes with the current ID format since it only incudes the `~~~` underlined part.

### Solution
Make the ID store the episode link fully, and use the episode list on the page to navigate around. Proposed format:

`<source:id#ep>`
Where:
- Source: a website name abbreviation followed by a version. E.g: `<GLP-1:...>`, `-1` is a version number to make sure the new format can be distingueshed from the old format and an error is then detectable.
- ID: The same as the `~~~` underlined part in the links above. `<...:pokemon-xy-z#...>`
- Ep: Episode number at the end of the link. E.g: `<...#12>`

Example ID: `<GLP-1:pokemon-xy-z#30>`.

For commands that need an episode in the series like `watch <id> <ep>` the episode should still be supplied since changing the episode in the identifier could yield an invalid link.

Such link encoding cautions should also be applied to any other sources.

### Version
0.1.1

**Comment by m-kuzmin** [comment]
I am currently implementing this.

**Comment by m-kuzmin** [comment]
Hopefully fixed by [0d661b6](https://github.com/m-kuzmin/ani-tui/commit/0d661b63dd8d899b57753e9f6e3403ef59ad0cad) and will be added `v0.2.0`
